src/executables/master_heartbeat.c

```diff
@@ -4836,9 +4869,12 @@ hb_cluster_initialize (const char *nodes, const char *replicas)
   struct sockaddr_in udp_saddr;
   char host_name[CUB_MAXHOSTNAMELEN];
 
-  if (nodes == NULL)
+  if (nodes == NULL || nodes[0] == '\0')
```

Comment:
How about removing this if statement and delegating the responsibility to hb_cluster_load_group_and_node_list()?

Reply:
thank you, it is good point.
I will accept your comment.